.changeset/early-forks-change.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sketch-hq/sketch-file-format': minor
```

Comment:
Something that's not quite obvious, but because the TypeScript types are a developer dependency instead of a direct dependency of the final build product (the TypeScript types package) we [must include the package specifically](https://github.com/sketch-hq/sketch-document#release). 

Only then will changesets also release an updated TypeScript types packages generated from the updated schema package. 

```suggestion
'@sketch-hq/sketch-file-format': minor
'@sketch-hq/sketch-file-format-ts': minor
```